samples/language/Ranges.qs

```diff
@@ -0,0 +1,49 @@
+/// # Sample
```

Comment:
Could you please merge this sample with the `Range.qs` sample taking into consideration the comments below?